src/reduced_basis/rb_evaluation.C

```diff
@@ -1054,19 +1061,19 @@ void RBEvaluation::read_in_vectors_from_multiple_files(System & sys,
   Xdr header_data(file_name.str(),
                   read_binary_vectors ? DECODE : READ);
 
-  // set the version number in header_data from io_version_string
-  // (same code as in EquationSystemsIO::_read_impl)
-  std::string io_version_string = get_io_version_string();
-  std::string::size_type lm_pos = io_version_string.find("libMesh");
-  std::istringstream iss(io_version_string.substr(lm_pos + 8));
+  // Set up the header info (based on EquationSystems::_read_impl).
+  std::string version;
+  header_data.data(version);
+  std::string::size_type lm_pos = version.find("libMesh");
```

Comment:
I know it's not really part of the patch, but the `string::find()` call without a corresponding `npos` check and the hard-coded 8 seems a little clunky to me.
```
const std::string label = "libMesh-";
std::string::size_type lm_pos = version.find(label);
// check for lm_pos==std::string::npos
std::istringstream iss(version.substr(lm_pos + label.size()));
```